docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -21,10 +21,55 @@ set up in accordance to these guides for either [Local Mode](Running-Alluxio-Loc
 
 Please [Download Hive](http://hive.apache.org/downloads.html).
 
-## Configuration
+To run Hive on Hadoop MapReduce, please also follow the instructions in
+[running MapReduce on Alluxio](Running-Hadoop-MapReduce-on-Alluxio.html) to make sure Hadoop
+MapReduce can run with Alluxio.
 
-Apache Hive allows you to use Alluxio through a generic file system wrapper for the Hadoop file system.
-Therefore, the configuration of Alluxio is done mostly in Hive and its under computing frameworks.
+## Use Alluxio as External Table Storage
```

Comment:
In the intro, you should specify that there are 2 ways to run hive on Alluxio. Also, is one method recommended?